src/mongocxx/test/collection.cpp

```diff
@@ -706,6 +741,24 @@ TEST_CASE("CRUD functionality", "[driver::collection]") {
         assert_contains_one("bar");
         assert_contains_one("quux");
     }
+
+    SECTION("distinct with collation", "[collection]") {
+        auto doc = document{} << "x"
+                              << "foo" << finalize;
+
+        auto predicate = document{} << "x"
+                                    << "FOO" << finalize;
+
+        auto distinct_opts = options::distinct{}.collation(case_insensitive_collation.view());
+
+        auto distinct_results = coll.distinct("foo", predicate.view(), distinct_opts);
```

Comment:
Should this be `coll.distinct("x",predicate.view(), distinct_opts);` since the field is `x`?